keras_cv/layers/preprocessing/random_flip.py

```diff
@@ -152,6 +155,9 @@ def augment_bounding_boxes(
                 "`RandomFlip(bounding_box_format='xyxy')`"
             )
 
+        if isinstance(bounding_boxes, tf.RaggedTensor):
```

Comment:
Why does this need to be made dense here?

Reply:
Getting the shape of a RaggedTensor is inconsistent, and it doesn't work if we don't make the tensor dense here.  The tensor is always dense at this point.

In a previous iteration this was required - but I just tested without it and it works now.  The base layer handles this now.